chemprop/uncertainty/uncertainty_calibrator.py

```diff
@@ -615,6 +622,285 @@ def apply_calibration(self, uncal_predictor: UncertaintyPredictor):
         return uncal_preds.tolist(), cal_preds.tolist()
 
 
+class ConformalCalibrator(UncertaintyCalibrator):
+    """
+    Conformal Calibrator.
+    """
+    @property
+    def label(self):
+        return f"{self.uncertainty_method}_conformal"
+
+    def raise_argument_errors(self):
+        super().raise_argument_errors()
+        if self.dataset_type != "classification":
+            raise ValueError(
+                "Conformal is only implemented for classification dataset types."
+            )
+
+    def calibrate(self):
+        uncal_preds = np.array(
+            self.calibration_predictor.get_uncal_preds()
+        )  # shape(data, tasks, num_classes)
+        targets = np.array(self.calibration_data.targets(), dtype=float)  # shape(data, tasks)
```

Comment:
I think this is actually intended to be multi-label classification? If the shape is `[num_examples, num_tasks]` then it's really performing predictions of {0, 1} for num_tasks different targets. In this case, you need to do multi-label conformal (https://arxiv.org/abs/2004.10181). 

Let's talk about this later.